packages/driver/cypress/e2e/commands/storage.cy.ts

```diff
@@ -406,6 +410,18 @@ describe('src/cy/commands/storage', () => {
           expect(lastLog.get('snapshots')[0]).to.be.an('object')
         })
       })
+
+      it('consoleProps includes the empty storage yielded', () => {
+        const ls = cy.state('window').localStorage
+
+        cy.clearLocalStorage().then(() => {
+          const consoleProps = logs[1].get('consoleProps')()
+
+          expect(consoleProps.name).to.equal('clearLocalStorage')
+          expect(consoleProps.type).to.equal('command')
+          expect(consoleProps.props.Yielded).to.deep.equal(ls)
+        })
```

Comment:
This is me trying to verify my assumption around what clearLocalStorage yields in the consoleProps (noted below more clearly as to why). There wasn't a test around this. Read comments below first.